test_cli.py

```diff
@@ -1434,6 +1434,10 @@ def compare_output_across_instances(command):
         _run_command([cl, 'rm', '-d', uuid])  # Keep only metadata
         _run_command([cl, 'add', 'bundle', uuid, '--dest-worksheet', remote_worksheet])
 
+        # Create at local, transfer to remote (non-terminal state bundle)
```

Comment:
This comment confused me a bit—could you clarify that you're making the bundle locally, but you want to ensure that it _isn't_ transferred to remote (since it's non-terminal)?

Reply:
@nelson-liu I have updated the tests and added more comments. Let me know if you think this is clear.